index.html

```diff
@@ -184,7 +184,7 @@ <h4 style="font-size: 18px!important">Service One</h4>
 
 
               <!-- Item 2 -->
-              <div onclick="location.href='services.html';" style="cursor:pointer;">
+              <div onclick="location.href='services.html#service-2';" style="cursor:pointer;">
```

Comment:
Can you make this `<a>` instead of `<div>`? Then you could just do `<a href="..."> ... </a>`. It would be better for accessibility reasons

Reply:
(and do the same for all 3 Service cards)